article/_posts/2020-08-31-TemporalConsistencyObjectives.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: review
+title: "Temporal Consistency Objectives Regularize the Learning of Disentangled Representations"
+tags: deep-learning segmentation semi-supervised autoencoder VAE
+author: "Nathan Painchaud"
+cite:
+    authors: "Gabriele Valvano, Agisilaos Chartsias, Andrea Leo, Sotirios A. Tsaftaris"
+    title:   "Temporal Consistency Objectives Regularize the Learning of Disentangled Representations"
+    venue:   "MICCAI 2019 (DART)"
+pdf: "https://arxiv.org/pdf/1908.11330.pdf"
+---
+
+
+# Highlights
+- Modality invariant transformer that provides a self-supervised signal and is used to predict future anatomical factors
+from current ones
+- State of the art performance compared to semi-supervised segmentation made by SDNet (from a previous paper by the same
+group[^1])
+- Preliminary results of cardiac temporal synthesis
+
+
+# Introduction
+The authors state that temporal correlations use in (cardiac) medical imaging has been limited to an additional
+signal/supervision for segmentation tasks. They propose to use spatiotemporal dynamics to improve disentangled
+representations.
+
+
+# Methods
+
+## Spatial Decomposition Network (SDNet)
+> The SDNet framework is an attempt to decouple anatomical factors from their appearance towards more explainable
+> representations.
+
+The SDNet can be seen as an autoencoder that learns multiple subspaces, namely:
+- $$s = f_A (x)$$: a multi-channel output of binary maps, representing the anatomical components
```

Comment:
how is s_t computed?  Quickly looked in the paper and did not found any mention of it.  Is it just a threshold?